providers/capz/standard/capz_test.go

```diff
@@ -23,7 +23,8 @@ type ClusterValues struct {
 
 var _ = Describe("Common tests", func() {
 	common.Run(&common.TestConfig{
-		BastionSupported: true,
+		BastionSupported:     true,
+		ExternalDnsSupported: false,
```

Comment:
We can enable it when this is merged https://github.com/giantswarm/default-apps-azure/pull/150

Reply:
It is merged but I imagine a new release needs generating first, yeah? Should we get that done now and remove this config flag?

Reply:
The config flag will still be needed, just enabling it `ExternalDnsSupported: true`. We still need to disable it for the non-cloud providers.